django/test/runner.py

```diff
@@ -470,6 +499,10 @@ def add_arguments(cls, parser):
             '--exclude-tag', action='append', dest='exclude_tags',
             help='Do not run tests with the specified tag. Can be used multiple times.',
         )
+        parser.add_argument(
+            '--pdb', action='store_true',
+            help='Runs the PDB debugger on error or failure.'
```

Comment:
Might want to avoid mentioning `PDB` since `ipdb` will be used if present?

Reply:
I've moved it to say pdb or ipdb instead, figure it's nice to be explicit.